### PR TITLE
[eclipse/xtext#1272] Set Xtext bootstrap 2.15

### PR DESCRIPTION
**Pull request by kthoms**
Refactored to use xtext_bootstrap property from versions.gradle
Changed target definition.

Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
Please make sure you remove the workarounds for the signature problem too.
Please let Arne or Holger review your prs

**Comment by cdietrich** [comment]
Qrrrggggg the signature problem workaround is still in place

**Comment by holgerschill** [comment]
Removing the Workaround should be another ticket. You don't think so?

**Comment by cdietrich** [comment]
No it is the same. Cause using Xtext 215 solves this issue. And if it is a separate ticket that ticket should have been created and the original one should have been adopted

**Comment by holgerschill** [comment]
You mean the explicit version range for some things?

**Comment by cdietrich** [comment]
no i mean this:  gradle/bootstrap-setup.gradle changed by https://github.com/eclipse/xtext-core/commit/0277b89235d6aa30682ca1f6bb159a7cf5333f9a#diff-5823690d259125c963d05526d1cdd7b9
similar changes are in place for all projects

**Comment by kthoms** [comment]
We'll remove the workarounds with separate PRs for issue #1305, OK?

**Comment by kthoms** [comment]
Removed the workarounds

File: org.eclipse.xtext.testlanguages/build.gradle [hunk 1]
```diff
@@ -4,9 +4,7 @@ dependencies {
 	compile project(':org.eclipse.xtext')
 	compile project(':org.eclipse.xtext.testing')
 	compile "org.eclipse.platform:org.eclipse.equinox.common:$versions.equinoxCommon"
-	compile("org.eclipse.platform:org.eclipse.core.runtime:$versions.eclipseCore") {
-		exclude group: "org.eclipse.platform", module: "org.eclipse.equinox.common"
-	}
+	compile "org.eclipse.platform:org.eclipse.core.runtime:$versions.eclipseCore"
```
Comment (cdietrich) [inline org.eclipse.xtext.testlanguages/build.gradle]: No karsten this is still needed. Only the bootstrap thing should go

Comment (kthoms) [reply org.eclipse.xtext.testlanguages/build.gradle]: OK, will update

Comment (cdietrich) [reply org.eclipse.xtext.testlanguages/build.gradle]: That affects all but bootstrap

Comment (cdietrich) [reply org.eclipse.xtext.testlanguages/build.gradle]: still needs change

**Comment by kthoms** [comment]
updated

**Pull request merged**